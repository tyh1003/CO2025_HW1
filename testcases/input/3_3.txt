10 
45 6 16 82 46 56 92 10 3 33 
