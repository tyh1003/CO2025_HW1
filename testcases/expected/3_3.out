3 6 10 16 33 45 46 56 82 92 
